src/weiroll/index.ts

```diff
@@ -9,6 +9,7 @@ export enum WeirollCommandFlags {
   CALL = 1,
   STATICCALL = 2,
   CALL_WITH_VALUE = 3,
+  // eslint-disable-next-line @typescript-eslint/no-duplicate-enum-values
   CALLTYPE_MASK = 3,
```

Comment:
Isn't this a bug?

Reply:
I'm not sure, but this is how it is in the original lib:
https://github.com/weiroll/weiroll.js/blob/main/src/planner.ts#L74